### PR TITLE
Update Readme - Describe how to specify config file

### PR DESCRIPTION
**Pull request by vukanac**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,3 +72,7 @@ As of version 0.4 phpDox requires an xml configuration file. In case a project y
 Sample invocation to parse and generate output based on the default phpdox.xml configuration file
 
     phpdox
+
+You can specify configuration file
```
Comment (theseer) [inline README.md]: Nice idea, please fix the english though ;)

Comment (vukanac) [inline README.md]: Any help with English, pls? This pass correctly with Spell checker. I don't know what to change.
I can put this line in README:

....
Sample invocation to parse and generate output based on the default phpdox.xml configuration file

    phpdox

or you can tell `phpdox` what configuration file to use by calling switch `--file` or in short

    phpdox -f path/to/phpdox.xml

Comment (theseer) [inline README.md]: That's way better :)

Comment (vukanac) [inline README.md]: Updated. And created/updated pull request.

**Pull request merged**